packages/react-radio/src/components/Radio/Radio.test.tsx

```diff
@@ -11,14 +10,56 @@ describe('Radio', () => {
     primarySlot: 'input',
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
+  it('renders a default state', () => {
+    render(<Radio />);
+    expect(screen.getByRole('radio')).toBeTruthy();
+  });
 
-  beforeEach(() => {
-    resetIdsForTests();
```

Comment:
Just to be safe can you restore the resetIdsForTests()?

Reply:
What's the reason to keep it? The only reason it was necessary before was for snapshots, but I removed the snapshot tests. The rest of the tests don't (and shouldn't) rely on a specific value of a generated ID.

Reply:
I guess I was thinking general safety/consistency? But searching the code it seems like we're very inconsistent with usage today (sometimes components use it even without snapshots, sometimes not).

Reply:
I think I'd prefer to remove it. I'd rather not have unnecessary code when the tests should not be relying on the stability of the IDs.